public/stylesheets/styles.css

```diff
@@ -0,0 +1,121 @@
+.btn-spacing {
+  margin: 5px;
+}
+
+.btn-window {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 15px;
+  width: 15px;
+  border-radius: 50%;
+  font-family: 'Roboto', sans-serif;
+}
+
+.btn-close {
+  background-color: #fc5855;
+}
+
+.btn-min {
+  background-color: #fec141;
+}
+
+.btn-open {
+  background-color: #35cb4b;
+}
+
+// .btn-close:hover {
+//   font-family: 'Roboto', sans-serif;
+// }
+
+// .btn-min:hover {
+//   font-family: 'Roboto', sans-serif;
+// }
+
+// .btn-open:hover {
+//   font-family: 'Roboto', sans-serif;
+// }
+
+.calculator-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #8E8E8E;
+  width: 100%;
+  height: 700px;
+}
+
+.calculator-font {
+  font-family: 'Roboto', sans-serif;
+  font-size: 25px;
+  align-self: center;
+  text-align: center;
+}
+
+.calculator-operations-font {
+  font-family: 'Roboto', sans-serif;
+  font-size: 25px;
+  color: #fff;
+}
+
+.calculator-body {
+  border: 1px solid #8E8E8E;
+  border-radius: 6px;
+  box-shadow: 10px 11px 40px 10px rgba(126, 126, 126, .6);
+  width: 300px;
+  height: 400px
+}
+
+.calculator-row {
+  display: flex;
+  width: 300px;
+  height: 60px;
+}
+
+.calculator-window {
+  display: flex;
+  background-color: #969696;
+  border: #8E8E8E;
+  width: 300px;
+  height: 100px;
+}
+
+.calculator-nums {
+  width: 300px;
+  height: 300px;
+}
+
+.calculator-button:hover, .calculator-button:active {
+  opacity: 0.8;
+  background-color: rgba(0, 0, 0, 0.5);
+  transition-timing-function: ease;
+  transition-duration: 100ms;
+  display: inline-flex;
+  justify-content: center;
+  width: 75px;
+  height: 58px;
+  border: 1px solid #8E8E8E;
```

Comment:
I don't see why you want to change the height, width, border or display properties on hover. I think all you need to change on hover is the background and border colors. 

Generally "transition" related properties would be placed on the `.calculator-button{ … }` and then the changes on `.calculator-button:hover{ … }`. Something like this:

```css
.calculator-button{
  display: inline-block;
  height: 20px;
  width: 20px;
  border: 1px solid grey;
  background-color: #ffeeff;
  transition: border-color 200ms, background-color 200ms;
}

.calculator-button:hover{
  border-color: #e2e2e2;
  background-color: #ee22ff
}
```